process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+# Becoming an OpenSSF Project
+
+The OpenSSF's mission is to inspire and enable the community to secure the open source software we all depend on. OpenSSF's top priority is the security of existing open source projects however, there are missing pieces in the tooling needed to improve open source project security. These pieces are the projects OpenSSF aims to steward and support.
+
```

Comment:
"projects" is used a couple different ways above that could be clarified.

Reply:
Addressed in PR#117.

Reply:
Since PR#117 is landed, should we update and resolve this?

Reply:
> Since PR#117 is landed, should we update and resolve this?

This is what I expected would happen. I don't have the right privileges for this so someone who does should mark it resolved along with the many others.